src/platform/packages/shared/kbn-unified-metrics-grid/src/store/index.ts

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Q: Do we need the Redux store because we use it in Discover? Maybe we can refactor the state to use the React Query State in the future, as we already have it for the APIs 🤔 But it could be that it won't be possible because of Discover, so we would have a mix, probably

Reply:
I think for now we'll have both state managements because we still can't hook some of the grid's internal state into Discover